aip/general/0134.md

```diff
@@ -103,6 +103,9 @@ message UpdateBookRequest {
   - The field **must** be optional, and the service **must** treat an omitted
     field mask as an implied field mask equivalent to all fields that are
     populated (have a non-empty value).
+    - Rationale is that omission of `update_mask` is equivalent on it being
```

Comment:
Rather than include this here, I'd suggest adding a rationale section separately.

Reply:
Indeed, and I'd take a glance at the guidance in AIP-8 for this section:
![image](https://github.com/user-attachments/assets/084c7802-2b07-4f16-8db2-7b5bcc8dd46c)